multiapps-controller-web/src/main/java/org/cloudfoundry/multiapps/controller/web/Messages.java

```diff
@@ -35,7 +35,7 @@
     public static final String ALM_SERVICE_ENV_INITIALIZED = "Deploy service environment initialized";
     public static final String CANNOT_AUTHENTICATE_WITH_CLOUD_CONTROLLER = "Cannot authenticate with cloud controller";
     public static final String ASYNC_DATABASE_CHANGES_WILL_NOT_BE_EXECUTED_ON_THIS_INSTANCE = "Async database changes will not be executed on instance {0}.";
-    public static final String STORING_TOKEN_FOR_USER_0_WITH_EXPIRATION_TIME_1 = "Storing token for user \"{0}\" with expiration time: {1} seconds";
+    public static final String STORING_TOKEN_FOR_USER_0_WHICH_EXPIRES_AT_1 = "Storing token for user \"{0}\" which expires at: {1}";
```

Comment:
DId you validate how the log will be shown?
I mean when we log the message, we will have timestamp of the log and expires at value must be in the same timezone.
Otherwise we could have inconsistency and the debugging will be harder

Reply:
It matches the value stored in the database (UTC)